pkg/tenant/organizations.go

```diff
@@ -72,3 +73,25 @@ func (s *Server) OrganizationDetail(c *gin.Context) {
 	}
 	c.JSON(http.StatusOK, out)
 }
+
+// Helper to fetch the orgID from the gin context. This method also logs and returns
+// any errors to allow endpoints to have consistent error handling.
```

Comment:
Do we have a follow on story to take care of the tech debt related to this?

Reply:
I was unable to find one.